src/chrome/content/rules/Dmm.co.jp.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="Dmm.co.jp">
+	<target host="dmm.co.jp" />
+	<target host="www.dmm.co.jp" />
+	<target host="avatar-contents.dmm.co.jp" />
+	<target host="display.dmm.co.jp" />
+	<target host="games.dmm.co.jp" />
+	<target host="pics.dmm.co.jp" />
+	<target host="p.dmm.co.jp" />
+
+	<rule from="^http://dmm\.co\.jp/"
```

Comment:
Have you found cases where this rule was needed on top of the trivial rule below? They seem to have a directive already for this domain to redirect to the "www" iteration.

Reply:
@zoracon Are you sure that https://dmm.co.jp/ and  https://dmm.com/ are accessible there? I can't access them. Please make sure that your browser is not automatically adding `www`.